packages/directory/addon/components/dir-item-name-cell.js

```diff
@@ -35,26 +28,23 @@ class DirItemNameCell extends Component {
   /**
    * @property {String} itemId - the id of the model or the tempId of an unsaved model
    */
-  @readOnly('value.modelId')
-  itemId;
+  get itemId() {
+    return get(this, 'args.value.modelId');
```

Comment:
is `value` required?

Reply:
Hmm?

Reply:
Could this be `this.args.value.modelId`?